README.md

```diff
@@ -6,8 +6,8 @@ This is an MSBuild SDK for [PascalABC.NET][pascalabc.net] programming language.
 NuGet
 -----
 
-`FVNever.PascalABC.NET.Compiler`: [![NuGet Package][nuget.compiler.badge]][nuget.compiler.package]
-`FVNever.PascalABC.NET.SDK`: [![NuGet Package][nuget.sdk.badge]][nuget.sdk.package]
+- `FVNever.PascalABC.NET.Compiler`: [![NuGet Package][nuget.compiler.badge]][nuget.compiler.package]
+- `FVNever.PascalABC.NET.SDK`: [![NuGet Package][nuget.sdk.badge]][nuget.sdk.package]
```

Comment:
Just suggesting, if you want to eliminate badges aligned to the top of the line, you could use`<img>` with a `valign` attribute instead of a Markdown-style image: 

 ```suggestion
- `FVNever.PascalABC.NET.Compiler`: [<img valign="middle" src="https://img.shields.io/nuget/v/FVNever.PascalABC.NET.Compiler">][nuget.compiler.package]
- `FVNever.PascalABC.NET.SDK`: [<img valign="middle" src="https://img.shields.io/nuget/v/FVNever.PascalABC.NET.SDK">][nuget.sdk.package]
```

<details>
  <summary>Before</summary>
<img src="https://user-images.githubusercontent.com/20597871/190906310-3589dad6-0f3f-45c6-9502-26820508b5e3.png">
</details>

<details>
  <summary>After</summary>
<img src="https://user-images.githubusercontent.com/20597871/190906539-cba74f4e-4cbe-4bf3-bdf1-450db12986c2.png">
</details>


Unfortunately, inside the `src` attribute, you couldn't use reference-style links and you need to hardcode the `<img>` source. But the NuGet link could be a reference-style link without any issues

Reply:
Thanks!